### ISSUE TITLE
Accessing env variables

### ISSUE THREAD

**Issue by ricardofbarros** [notablemind/jupyter-nodejs#5]
Accessing `process.env.{variable}` kills the kernel.

For example:

![screen shot 2015-05-27 at 18 45 57](https://cloud.githubusercontent.com/assets/6867996/7843297/a70735a4-04a0-11e5-9006-b79cbd71fa8d.png)

**Comment by jaredly** [comment]
wow.

**Comment by ricardofbarros** [comment]
But I can perfectly access `process.env` and it will return the object

**Comment by jaredly** [comment]
right? that's weird

**Comment by ricardofbarros** [comment]
Yeah very weird, I will try to look into this, maybe they are using `Object.defineProperty` API.

**Comment by jaredly** [comment]
I think it's a problem with `Contextify`
